package.json

```diff
@@ -7,7 +7,7 @@
     "lint:js": "eslint js/*.js",
     "lint:tests": "eslint test/*.js",
     "lint:html": "htmlhint index.html",
-    "lint:css": "csslint --format=compact --warnings=box-model,ids css/*.css",
+    "lint:css": "csslint --format=compact --warnings=box-model,ids css/",
```

Comment:
I was getting "file not found"  with csslint using the global pattern  "*.css", having this way make csslint check every file in the folder not only css, you can revert it, although it very obvious that a css folder should contain css files